common/religion/religions/fire_group.txt

```diff
@@ -46,8 +46,8 @@
 
 	# Placeholder
 	traits = {
-		virtues = { wrathful zealous patient }
-		sins = { calm cynical impatient }
+		virtues = { wrathful zealous brave }
+		sins = { calm cynical craven }
```

Comment:
I would suggest changing 2 traits.
So here is our reference.

![image](https://user-images.githubusercontent.com/46384992/144747213-093c9337-7531-45dd-bf4c-de23dfae4ce8.png)

I wouldn't say passion is bravery.
I would say passion is either Diligent or Ambitious.
And fury is Wrathful and Impatient.

So virtues will be wrathful, impatient, diligent/ambitious.
Sins will be calm, patient, lazy/content.